tests/nightly/model_backwards_compatibility_check/common.py

```diff
@@ -42,7 +42,7 @@
 
 # get the current mxnet version we are running on
 mxnet_version = mx.__version__
-model_bucket_name = 'mxnet-model-backwards-compatibility-models'
+model_bucket_name = 'mxnet-ci-prod-backwards-compatibility-models'
```

Comment:
This bucket name change also has to be made in upload_models_to_s3.sh.

Done that in 33096c02f56186464b34ba6b88480290eb89263f